BrownianMotion/Continuity/KolmogorovChentsov.lean

```diff
@@ -14,6 +14,14 @@ import BrownianMotion.Continuity.IsKolmogorovProcess
 open MeasureTheory
 open scoped ENNReal NNReal
 
+section aux
+
+theorem lintegral_eq_zero_of_zero_ae {α : Type*} [MeasurableSpace α] {μ : Measure α}
+    {f : α → ℝ≥0∞} : f =ᵐ[μ] 0 →  ∫⁻ a, f a ∂μ = 0 :=
+  fun h ↦ (lintegral_congr_ae h).trans lintegral_zero
```

Comment:
That's just the `mpr` half of [MeasureTheory.lintegral_eq_zero_iff](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Integral/Lebesgue/Basic.html#MeasureTheory.lintegral_eq_zero_iff)?

Reply:
Ah right, I did not see that difference. I agree then.